tests/test_server.py

```diff
@@ -0,0 +1,105 @@
+import asyncio
+import unittest
+import json
+
+from unittest import mock
+
+
+class TestServer(unittest.TestCase):
+    MockedRequestHandler = None
+    srv = None
+    m = None
+    m_eof = None
+
+    def setUp(self):
+        with mock.patch('builtins.open', mock.mock_open(), create=True):
+            with mock.patch('pickle.load', mock.Mock(), create=True):
+                import server
```

Comment:
When I make import  at the top of the file, it tries to open file and fails, that's why I mock open function and import server after mock patching. How can I make it better?